app/components/pathway/PathwayCard.tsx

```diff
@@ -61,31 +61,67 @@ const PathwayCard = ({ title, department, coursesIn }: IPathwaySchema) => {
     console.log(courses);
   }
   */
+
+  let completed = inPathway.filter((course) => course.status === "Completed");
+  let inProgress = inPathway.filter((course) => course.status === "In Progress");
+  let planned = inPathway.filter((course) => course.status === "Planned");
+  let total_size = completed.length + inProgress.length + planned.length;
+  let max_size = 3; // Change this variable to change pathway card course limit
+  if (total_size > max_size) {
+    if (completed.length > max_size) {
+      completed = completed.slice(0, max_size);
+    }
+    let current_size = completed.length + inProgress.length;
+    if (current_size > max_size) {
+      inProgress = inProgress.slice(0, max_size - completed.length);
+    }
+    current_size = completed.length + inProgress.length;
+    total_size = completed.length + inProgress.length + planned.length;
+    if (total_size > max_size) {
+      planned = planned.slice(0, max_size - current_size);
+    }
+  }
```

Comment:
Is there a reason to not sort the whole thing based on the status then slice at the end?

Reply:
You're probably right, I just didn't think of it because I would have to map each status tag to a numerical value.